README.md

```diff
@@ -8,157 +8,147 @@
 [![Coverage Status](https://coveralls.io/repos/github/Unleash/unleash-edge/badge.svg?branch=main)](https://coveralls.io/github/Unleash/unleash-edge?branch=main)
 ![downloads](https://img.shields.io/crates/d/unleash-edge.svg)
 
-> Warning: Unleash Edge requires Unleash v4.15.0 or higher
+- [Overview](#overview)
```

Comment:
Not sure if we want this. I think it helps navigate, but it's quite big.. 🤔

Reply:
Yeeee... maybe? I'm leaning slightly towards "we want this"